pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -300,7 +300,9 @@ clone_user_rev() {
     local rev="${3:-HEAD}"
 
     if [ -n "$fetchLFS" ]; then
-        HOME=$TMPDIR
+        tmpHomePath="$(mktemp -d "${TMPDIR:-/tmp}/nix-prefetch-git-tmp-home-XXXXXXXX")"
+        exit_handlers+=(remove_tmpHomePath)
```

Comment:
Can we execute the command directly?

Reply:
IMHO, doing changes like that is orthogonal to this patch series. Other places use exit_handlers like that.

Reply:
What patch series? I only see 3 commits which only do this once.

Reply:
I meant the patches (commits) in this PR. The script already registers cleanup commands in the exit_handlers array, I just continued to use the existing mechanism.